PostHog/PostHogMaskViewModifier.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  PostHogMaskViewModifier.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 09/10/2024.
+//
+
+#if os(iOS) && canImport(SwiftUI)
+
+    import SwiftUI
+
+    public extension View {
+        func postHogMask(_ value: Bool = true) -> some View {
```

Comment:
Do we need the `value` if there's no way to force it not to be masked? probably better to leave without it

Reply:
makes sense, not sure `isActive` is clear but its already better than `value` that has no meaning at all in this context, maybe `maskIt: Bool = true` is more clearer, we can inspire from any other swift docs btw
https://developer.apple.com/documentation/swiftui/view/mask(_:) they use `mask` only but its an object